src/Disks/IDisk.h

```diff
@@ -363,6 +363,9 @@ class IDisk : public Space
 
     virtual bool isWriteOnce() const { return false; }
 
+    /// Whether this disk support mutations.
+    virtual bool isMutable() const { return true; }
```

Comment:
May be better to explicitly name it `supportsHardlinks`? Because `isMutable` is kind of the opposite of `IDisk::isReadOnly`, so might be confusing